data/mods/PKs_Rebalance/pk_critter_misc.json

```diff
@@ -630,8 +630,7 @@
             "type": "bite",
             "cooldown": 30,
             "accuracy": 7,
-            "no_infection_chance": 3,
-            "armor_multiplier": 0.5
```

Comment:
They are only valid within a `damage_type`, for example `damage_max_instance`. They aren't read by the bite actor at all otherwise.